repose-aggregator/core/repose-core/build.gradle

```diff
@@ -15,6 +15,8 @@ dependencies {
     compile project(":repose-aggregator:components:services:http-client-service:http-client-service-api")
     compile project(":repose-aggregator:components:services:datastore-service:datastore-service-api")
     compile project(":repose-aggregator:components:services:atom-feed-service:atom-feed-service-impl")
+    compile project(":repose-aggregator:components:services:container-configuration-service:api")
```

Comment:
you shouldn't need the api, unless you are directly using the api contract in some way?